.github/workflows/integration.yml

```diff
@@ -2,10 +2,15 @@ name: integration
 
 on:
   push:
-    branches: [ main ]
+    branches:
+      - '*'
+      - '!generate/aws-lc-*'
   pull_request:
     branches:
       - '*'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
```

Comment:
is `pull_request` populated on `push`-type events? i don't see any such fields in the [docs](https://docs.github.com/en/webhooks/webhook-events-and-payloads#push)...